src/RolloutScorer/RolloutScorer/RolloutScorer.cs

```diff
@@ -313,7 +313,8 @@ public async Task<List<Issue>> GetRolloutIssuesFromGithubAsync()
         {
             SearchIssuesRequest searchIssuesRequest = new SearchIssuesRequest
             {
-                Created = new DateRange(RolloutStartDate, RolloutEndDate),
+                // use offsets here because the GitHub API gets mad about UTC specified with a +
```

Comment:
Wait, what?  This can't possibly be real... There is no way that there is an API that only works if you shift off of UTC by an hour west.

Reply:
If that _is_ true, did we file an issue with OctoKit to get it fixed?

Reply:
At least in the UI, this search seems to work fine:
https://github.com/dotnet/core-eng/issues?q=is%3Aissue+is%3Aopen+created%3A%3C2020-01-01T01%3A01%3A01.0000Z+, so UTC doesn't seem to be an issue.

Reply:
VS tells me the implementation is this:
```csharp
public DateRange(DateTimeOffset from, DateTimeOffset to)
{
   this.query = from.ToString("yyyy-MM-dd'T'HH:mm:sszzz", (IFormatProvider) CultureInfo.InvariantCulture) + ".." + to.ToString("yyyy-MM-dd'T'HH:mm:sszzz", (IFormatProvider) CultureInfo.InvariantCulture);
}
```
Which means it's not honoring the timezone at all, so this change should actually do anything (except add one to the HH part, I think)

Reply:
https://github.com/octokit/octokit.net/issues/2056 which was fixed in Feburary.  Let's put a "TODO" here and open a bug to remove this hack when we upgrade to a release with that fix in it. :-)

Reply:
yeah, take a look at the examples in the linked issue. I think the root bug is in the GitHub API (failing on offset of +0:00 but working with -0:00 and Z) and then OctoKit happens to send the UTC offset as +0:00. It's a wild bug but this workaround does make it work.

Reply:
to be clear, the specific bug only occurs if you specify this date range with a UTC offset, which the rollout scorer does bc the VM it runs on has a UTC local time. That's why this evaded me for so long; because my local time is -08:00 or -07:00 it always works on my local machine. I was able to repro it by using `.ToOffset(0)`.

Reply:
Did you try updating to 40 for that package? The bug was fixed and released several months ago, I think.

Reply:
Wow sorry I guess I didn't hit refresh before commenting; I didn't see the other comments after your first lol. I'll update and hopefully that fixes it!

Reply:
yep looks like that fixed it! thanks Chad!

Reply:
also fun fact @ChadNedzlek -- it does work in search, but if you try the same thing via the API it will break which honestly just raises more questions